src/test/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksTaxRuleDependencyServiceImplTest.java

```diff
@@ -96,5 +98,23 @@ void testBuildDateFromString() {
         
         assertEquals(expectedDate, actualDate);
     }
+    
+    @Test
+    void testPopulateOwnershipCodeLLC_C_Corp() {
+        PaymentWorksVendor pmwVendor = new PaymentWorksVendor();
+        pmwVendor.setRequestingCompanyTaxClassificationCode(PaymentWorksConstants.LLC_TAXED_AS_C_CORPORATION_TAX_CLASSIFICATION_INDICATOR);
+        VendorHeader vendorHeader = new VendorHeader();
+        taxRuleService.populateOwnershipCode(pmwVendor, TaxRule.NOT_INDIVIDUAL_US, vendorHeader);
+        assertEquals(VendorOwnershipTypeCodes.C_CORPORATION, vendorHeader.getVendorOwnershipCode());
+    }
+    
+    @Test
+    void testPopulatenOwnernshipCodeLLC_S_Corp() {
```

Comment:
This method has a few typos in it still.

Reply:
fixed